save-orchestrator-common/src/test/kotlin/com/saveourtool/save/orchestrator/utils/DockerClientTestConfiguration.kt

```diff
@@ -14,7 +14,7 @@ class DockerClientTestConfiguration {
     /**
      * @return [DockerClient]
      */
-    @Bean
+    @Bean(destroyMethod = "")
```

Comment:
?

Reply:
It's for tests. I re-used dockerClient fromt testcontainers and they prohibit to destroy it (there were warning message in logs)

Reply:
oh ok